internal/doc/butane.yaml

```diff
@@ -344,6 +344,11 @@ root:
               desc: sets the minimum number of pieces required to decrypt the device. Default is 1.
             - name: discard
               desc: whether to issue discard commands to the underlying block device when blocks are freed. Enabling this improves performance and device longevity on SSDs and space utilization on thinly provisioned SAN devices, but leaks information about which disk blocks contain data. If omitted, it defaults to false.
+            - name: cex 
+              desc: describes the IBM Crypto Express (CEX) card configuration for the luks device. 
+              children:
+                - name: enabled
+                  desc: whether or not to use a CEX secure key to encrypt the luks device.
```

Comment:
"whether or not to enable cex compatibility for luks. If omitted, defaults to false."

Reply:
Done

Reply:
Hi @prestist , 
The description is automatically taken from the field enabled from the Ignition. I tried to change in `butane.yaml` but `./generate`  creates the desc from the ignition.

Reply:
Right; sigh eventually we will want to change that. But not blocking I guess (must have missed it)

Reply:
Then we should update the description in Ignition before

Reply:
@travier , @prestist , A PR has been submitted to fix this.
https://github.com/coreos/ignition/pull/1930  Kindly review.